.docker/upstream.repos

```diff
@@ -0,0 +1,9 @@
+repositories:
```

Comment:
There already is a `moveit_drake.repos` file at the top level. Can probably just keep one of these 2 files.

Reply:
Actually inspired from [upstream.repos](https://github.com/moveit/moveit2_tutorials/blob/main/.github/upstream.repos) of moveit2_tutorials. While installing via docker, we don't need to clone repositories like moveit2 (they already installed in MoveIt's docker image) . In case of local installation, user need to clone additional repositories in addition of `upstream.repos`. Shortly, i have tried to support both local installation method and docker installation. But if you don't mind, we can only update main `moveit_drake.repos`

Reply:
Yes this seems to be the likely way to go forward. I am in favor of keeping both for now. This gives users the option of doing a single VCS import if working locally.

Could you add some instructions on README, under the local installation section to point users to this effect? May also be beneficial to specify that the build time is large in this case.